oat-sa/extension-tao-testqti
# Fix/MS-819/translate warning messages

**Related to task:**  https://oat-sa.atlassian.net/browse/MS-819
**Description:** Extend warning message with confirmations to perform the action.
**Requires:** 
- [x] Release FE package https://github.com/oat-sa/tao-test-runner-qti-fe/pull/344
- [x] Update package*.json files

**How to test:** 
- Create a test and activated the warnings in properties
<img width="374" alt="Screenshot 2021-01-05 at 17 08 13" src="https://user-images.githubusercontent.com/6339002/103669426-9df9dd80-4f78-11eb-950d-88721ee407a6.png">
- Login as TT and pass the test to see the warnings.
